deploy/charts/external-secrets/templates/servicemonitor.yaml

```diff
@@ -36,4 +36,13 @@ spec:
   - port: metrics
     interval: {{ .Values.serviceMonitor.interval }}
     scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
+    honorLabels: {{ .Values.serviceMonitor.honorLabels }}
+    {{- with .Values.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+      {{- tpl (toYaml . | nindent 6) . }}
```

Comment:
Why do you need to `tpl` here? I'm curious about the use-case of this.

Reply:
copy paste errors. fixed